docs/geedocs/5.2.2/answer/176676.html

```diff
@@ -6,6 +6,7 @@
   <div class="sidebar1">
 <p class="sidebar-toc"><a href="../answer/3481464.html">Before you configure Google Earth Enterprise Fusion</a></p>
 <p class="sidebar-toc"><a href="../answer/6008655.html">Running Fusion on a Machine with Multiple CPUs</a></p>
+<p class="sidebar-toc"><a href="../answer/7160003.html">Dynamically adjusting logging levels</a></p>
```

Comment:
These changes should be in the 5.2.4 documentation folder, since that's the version they'll be introduced in. We should also add a note to the release notes about the new feature.